client/src/views/MagicLinkPage.js

```diff
@@ -56,27 +59,27 @@ function MagicLinkPage({ isAuthenticated, email, secret, token, storageKeys }) {
   return (
     <>
       <TopNavbar isAuthenticated={isAuthenticated} />
-      <div className='wrapper'>
+      <div className="wrapper">
         <Container>
           <h1>Your Magic Link</h1>
           <p>Magic link explanation</p>
-          {(!token && !data && !loading && !email) && (
+          {!token && !data && !loading && !email && (
             <Form onSubmit={onSubmit}>
               <FormGroup>
-                <Label for='email'>E-mail</Label>
+                <Label for="email">E-mail</Label>
                 <Input
                   required
-                  type='email'
-                  name='email'
+                  type="email"
+                  name="email"
                   onChange={changeInput}
-                  placeholder='Your e-mail address so we can send your token'
+                  placeholder="Your e-mail address so we can send your token"
                 />
               </FormGroup>
-              <Button type='submit'>Send magic link</Button>
+              <Button type="submit">Send magic link</Button>
             </Form>
           )}
           {loading && <p>Sending email...</p>}
```

Comment:
There's a lot of just style noise with the ' -> " single to double quotes, have we set up any of the local pettier/eslint config files so this can just get standardized?

Reply:
I ran eslint from the client root which seems to be using `react-app` defaults... but not sure a configs are being changed by prefences in my machine.